Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -0,0 +1,2124 @@
+#import "WMFExploreViewController.h"
```

Comment:
Just checking - is `WMFExploreViewController.m` supposed to be here?

Reply:
nope, it should be gone

Reply:
good catch @montehurd 👍🙌  somehow it stayed in my folder even though Xcode wasn't seeing it 🤔